surveys/2021-trademark-survey/questions.md

```diff
@@ -0,0 +1,119 @@
+# Survey questions
+
+Whether or not you use Rust <https://rust-lang.org> today, we want to hear from
+you!
+
+The Rust Foundation has created this survey to help us design a trademark
+survey that best serves the users of the Rust programming language.
+
+The purpose of this questionnaire is to get directional information. For that
+reason, if you largely agree with an answer but not entirely, you should say
+you agree. There will be further refinements as the trademark guidelines are
+developed where we will be working out the nuances.
+
+We are referring to the words RUST and CARGO and the logos, so assume the
+question is about both the words and the logos. If the question is about only
+the logos, it will be called “the Rust/Cargo logo.”
+
+Unless you choose to enter your email, your answers will be anonymous. Any
+personal data you submit as a part of this survey will be handled in accordance
+with our policy as described in our Frequently Asked Questions:
+
+https://github.com/rust-community/team/wiki/State-of-the-Rust-Language-Community-Survey-FAQ
+
+We estimate it will take about ? minutes to complete.
+
+> TODOs:
+>
+> - Evaluate survey length
+> - Can the community survey FAQ be generalized for all surveys such as this or
+>   the trademark surveys?
+> - Resolve and close open issues related to this survey on `rust-lang/surveys`
+> - Are these questions being presented in the optimal order?
+
+## Trademark Survey
+
+### My personal belief about the Rust trademarks is:
+
+Pick as many as you agree with:
+
+- The Rust Foundation should be highly protectionist about them
+- Trademarks are different from copyrights, so it’s ok to exert some control over how people use them
+- The Rust Foundation should share trademarks as freely as it shares code
+
+> **justification**
+>
+> TODO
+
+### If someone creates a Rust package from the original source code without changing the code in any way, can they still call the compiled package “Rust”?
+
+- Yes
+- No
+
+> **justification**
+>
+> TODO
+
+### How much of the Rust language can someone change and still call it “Rust”?
+
+Pick as many as you agree with:
+
+- Not at all
+- Adding new functions/syntax but not removing anything or breaking anything
+- Fixed bugs and other minor changes are ok
+- They can change it any way they want
+
+> **justification**
```

Comment:
I am not quite clear on what this is going to be used for?

Reply:
Justification sections are meant to highlight why a question is being asked. This is only for survey creators to leave notes to future survey creators why certain decisions were made. This will not be shown in the actual survey itself.

Reply:
The justifications are also useful for identifying and pruning unnecessary questions.

Reply:
Thanks. Should we determine the justification now, before merging this pull request? Or can that be done after the fact?

Reply:
It can be done after the fact, absolutely no problem to split the changes up over multiple PRs